netlify.toml

```diff
@@ -1,147 +1,85 @@
-# Settings in the [build] context are global and are applied to all contexts
-# unless otherwise overridden by more specific contexts.
 [build]
-  # Directory to change to before starting a build.
-  # This is where we will look for package.json/.nvmrc/etc.
-  base = ""
-
   # Directory (relative to root of your repo) that contains the deploy-ready
-  # HTML files and assets generated by the build. If a base directory has
-  # been specified, include it in the publish directory path.
+  # HTML files and assets generated by the build
   publish = "website/build/testing-library-docs"
 
-  # Default build command.
-  command = "cd website; npm install; npm run format-docs; npm run build"
+  # Default build command
+  command = "npm i -g yarn; cd website; yarn; yarn format-docs; yarn build"
```

Comment:
Guess we'll need to change this once https://github.com/testing-library/testing-library-docs/pull/607 is merged..

Reply:
Done, we can revert this part

Reply:
```suggestion
  command = "cd website; npm install; npm run format-docs; npm run build"
```